traffic_ops/traffic_ops_golang/profileparameter/parameterprofilebyname.go

```diff
@@ -31,11 +31,16 @@ import (
 func GetProfileName(w http.ResponseWriter, r *http.Request) {
 	inf, userErr, sysErr, errCode := api.NewInfo(r, []string{"name"}, nil)
 	if userErr != nil || sysErr != nil {
-		api.HandleErr(w, r, inf.Tx.Tx, errCode, userErr, sysErr)
+		api.HandleDeprecatedErr(w, r, nil, errCode, userErr, sysErr, nil)
```

Comment:
this should have an alternative at `profiles/name/{name}/parameters`

Reply:
so this function is actually used for this endpoint and the `profiles/name/{name}/parameters` endpoint which is the alternative for this after its deprecated. we will have to split that up so that the alternative doesnt also get the deprecation notice